### DISCUSSION TITLE
VcSelectionIndicator选择器在VcLayerImagery和Primitive同时存在时，无法选中Primitive

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#588] [Q&A] [answered]
如题，在同一个地方同时存在VcLayerImagery和使用Cesium.Primitive生成的图元，VcSelectionIndicator只能选中VcLayerImagery，选不到Primitive，不知道是不是bug？还是有什么方法可以设置？

**Comment by zouyaoji** [comment, marked as answer]
可能是拾取函数先拿到了影像图层，可以尝试通过ref的方法getPickedFeatures拿到所有的拾取对象

**Reply by HHKMFZY** [reply]
selectionIndicatorRef?.value?.getPickedFeatures()?.value?.features可以拿到，感谢！
